.github/workflows/apiBreakTest.yml

```diff
@@ -0,0 +1,174 @@
+name: "Check API Breakage"
```

Comment:
it's not clear what the difference between "Check API Breakage" and "Swift API Digester is.  Can you provide a more descriptive name and/or provide some description via code comments?